tests/README.md

```diff
@@ -1,26 +1,52 @@
 ## TDX Tests
 
-The tests should be run locally on TDX host.
+This folder contains TDX tests.
 
-It is assumed that these commands are running on a TDX host
-that has been setup properly.
+### Pre-requisites
 
-The guest image must be available at the location : `checkbox-provider-tdx/data/tdx-guest.qcow2`
-The guest image must enable ssh server with password-based authentication for `root` user.
-The root user password must be `123456`
+- The tests must be executed a TDX host that has been setup properly.
 
-Run tests with pytest
+- The guest image must be available at the project folder with the name `tdx-guest.qcow2`
+  You can follow the instructions in the project README to create one guest image.
+  You can specify a path to the guest image with `TDXTEST_IMAGE_FILE` environment variable.
+
+- The guest image must enable ssh server with password-based authentication for `root` user.
+  The root user password must be `123456`
+
+### Run tests with pytest
+
+- Run sanity tests to check the TDX host setup:
+
+```
+$ cd tests
+$ sudo ./run.sh pytest bin/test_host_*.py
+```
+
+- Run sanity tests to check the TDX guest boot:
 
 ```
-$ ./run.sh pytest
+$ cd tests
+$ sudo ./run.sh pytest bin/test_guest_*.py bin/test_boot_*.py
```

Comment:
The boot tests pass but I do see a few errors/warnings:

```
qemu-system-x86_64: Cannot find TDX_METADATA_OFFSET_GUID
qemu-system-x86_64: failed to parse TDVF for TDX VM
```

Reply:
The errors/warnings are part of the expected outcome of the test. Here, we want to test a failure boot situation

Reply:
If it's expected, should we capture that output with pytest and assert that it contains these strings?

Reply:
We check only the action failed as expected and not the error output log. No need to check the error output here i think.